content/blog/2016/2016-07-04-ewm-alpha-version.adoc

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "GSoC: External Workspace Manager Plugin alpha version"
+tags:
+- pipeline
+- plugins
+- external-workspace-manager
+- gsoc
+author: alexsomai
+---
+
+Currently it's quite difficult to share and reuse the same workspace between multiple jobs and across nodes.
```

Comment:
If I add the first paragraph that references the original blog post, should I remove this paragraph? WDYT?